dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/LegacyTrnJourney.cs

```diff
@@ -152,27 +191,6 @@ AuthenticationState.TrnLookup switch
             _ => base.GetLastAccessibleStepUrl()
         };
 
-    protected async override Task<IActionResult> OnEmailVerifiedCore(User? user)
```

Comment:
This didn't seem to be used anywhere

Reply:
Once we've migrated everything over to using these journey the intentation was for the `Email` and `EmailConfirmation` pages to call into these methods.